sdk/documentintelligence/azure-ai-documentintelligence/samples/sample_analyze_custom_documents.py

```diff
@@ -89,6 +89,22 @@ def analyze_custom_documents(custom_model_id):
                     print(f"...{region.page_number}")
             for cell in table.cells:
                 print(f"...Cell[{cell.row_index}][{cell.column_index}] has text '{cell.content}'")
+
+    # Extract table cell values from "documents" field in result and print the name/value information
+    if result.documents:
+        for idx, doc in enumerate(result.documents):
```

Comment:
for idx, doc in enumerate(result.documents or []):

Reply:
fixed thx